staker-rewards/lib/numbers.js

```diff
@@ -0,0 +1,7 @@
+import BigNumber from "bignumber.js"
+
+export const decimalPlaces = 2
+export const noDecimalPlaces = 0
+
+export const shorten18Decimals = (value) =>
```

Comment:
Not related to this PR but... can we take the opportunity and use `shorten18Decimals` to fix the value displayed at the beginning of the script execution:
```
OUTPUT_MODE="text" ETH_HOSTNAME="..." node --experimental-json-modules rewards.js 1606928319 1607533119 3934656000000000000000000
(...)
Interval total rewards: 3934656000000000000000000 KEEP
```
?

(should be `3934656 KEEP`)